developer_manual/client_apis/WebDAV/comments.rst

```diff
@@ -0,0 +1,151 @@
+=========

+Comments

+=========

+

+Comments have a PHP API within OCP and also via WebDAV. Here is a very basic description of the WebDAV API.

+

+The Comments resource has an endpoint::

+

+   remote.php/comments/($OBJECTTYPE/$OBJECTID) [/$COMMENTID]

+

+The ObjectID endpoint accepts:

+

+================  ============  =========================================  =============

+Function          Action        URL                                        Remark

+================  ============  =========================================  =============

+create comment    POST          …/${entityType}/${entityId}/               in future maybe also this url with several IDs appended (== replies)

+update comment     PROPPATCH    …/${entityType}/${entityId}/ ${commentId}  to update the read mark, property name: {http://owncloud.org/ns}readMarker

+delete comment    DELETE        …/${entityType}/${entityId}/ ${commentId}

+list comments     PROPFIND      …/${entityType}/${entityId}                The Comments Manager returns back the whole comments. *Filter restrictions (limit, offset, datetime) can not be provided with PROPFIND.*

+list comments     REPORT        …/${entityType}/${entityId}                The Comments Manager returns back the whole comments. Filter restrictions (limit, offset, datetime) may be provided per request header.

+================  ============  =========================================  =============

+

+The CommentID endpoint accepts:

+

+* PROPPATCH to update the comment

+* DELETE to delete it

+* PROPFIND to list all properties

+

+

+Here is a full list of current properties::


```

Comment:
ah, i misunderstood what you meant with leaving away things. In fact, the whole namespace is of importance. Like in https://docs.nextcloud.com/server/16/developer_manual/client_apis/WebDAV/search.html 

Also the `self::PROPERTY_NAME_FOO`` needs to be resolved as it does not help here. self is a reference to a php class of which we do not know here and the other part is the name of a constant.